src/main/discord_bot.py

```diff
@@ -16,25 +16,21 @@ def __init__(self, *args, **kwargs):
 
         # TODO figure out a way to do this programatically so it can be stored in a config
         self.parrot_party = ParrotParty(self)
-        self.status_check = StatusCheck(self)
+        self.status_check = StatusCheck(self, kwargs['status_check_config'])
```

Comment:
Can't this just be `config['status.check.config']`?
```suggestion
        self.status_check = StatusCheck(self, config['status.check.config'])
```

Reply:
snake case is the pythonic way

also using module level variables to instantiate classes is bad practice. If DiscordBot were being used in another module, config would not exist and therefore it would always error